### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by frankenjoe**
image

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,6 +7,58 @@ The format is based on `Keep a Changelog`_,
...
+--------------------------
+
+* Added:
+  ``audbackend.Artifactory._use_legacy_file_structure()``,
+  ``audbackend.Backend.access()``,
+  ``audbackend.Backend.available()``,
+  ``audbackend.Backend.date()``,
+  ``audbackend.Backend.delete()``,
+  ``audbackend.Backend.owner()``
+  ``audbackend.FileSystem._use_legacy_file_structure()``
```
Comment (hagenw) [inline CHANGELOG.rst:20]: I would split this into two points:

```suggestion
* Added:
  ``audbackend.Backend.access()``,
  ``audbackend.Backend.available()``,
  ``audbackend.Backend.date()``,
  ``audbackend.Backend.delete()``,
  ``audbackend.Backend.owner()``
* Added:
  ``audbackend.FileSystem._use_legacy_file_structure()``
  to support file structure of existing repositories
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -7,6 +7,58 @@ The format is based on `Keep a Changelog`_,
...
+  (``audbackend.access()`` should be used instead)
+* Changed: ``audbackend.Backend.get_archive()``
+  and ``audbackend.Backend.put_archive()``
+  support same archive types as ``audeer.create_archive()``
+* Changed: ``audbackend.Backend.get_file()``
+  skips operation if file with same checksum exists on local file system
+* Changed: ``audbackend.Backend.get_file()`` uses a temporary directory
+  to avoid corrupted files if operation is interrupted
+* Changed: ``audbackend.Backend.get_file()``
+  and ``audbackend.Backend.put_file()`` raises ``IsADirectoryError``
```
Comment (hagenw) [inline CHANGELOG.rst:34]:
```suggestion
  and ``audbackend.Backend.put_file()`` raise ``IsADirectoryError``
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -7,6 +7,58 @@ The format is based on `Keep a Changelog`_,
...
+* Changed: ``audbackend.put_archive()`` raises ``NotADirectoryError``
+* Changed: make ``files`` an optional argument of
+  ``audbackend.Backend.put_archive()``
+* Changed: ``audbackend.Backend.put_file()``
+  passes checksum to implementation to avoid re-calculation
+* Changed: ``audbackend.Backend.join()`` and ``audbackend.Backend.split()``
```
Comment (hagenw) [inline CHANGELOG.rst:40]: Here the actual change is missing.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -7,6 +7,58 @@ The format is based on `Keep a Changelog`_,
...
+* Changed: ``audbackend.Backend.ls()`` returns list of ``(path, ext, version)``
+* Changed: ``audbackend.Backend.ls()`` accepts full path
+  check for invalid characters
```
Comment (hagenw) [inline CHANGELOG.rst:43]: I guess this does not belong to the line before it?

File: CHANGELOG.rst [hunk 5]
```diff
@@ -7,6 +7,58 @@ The format is based on `Keep a Changelog`_,
...
+* Changed: calculate checksum with ``audeer.md5()``
+* Changed: file structure on ``audbackend.FileSystem``
+  and ``audbackend.Artifactory`` from
+  ``/sub/file/1.0.0/file-1.0.0.txt``
+  to
+  ``/sub/1.0.0/file.txt``
+* Changed: remove ``ext`` argument
+* Changed: backend path must start with ``'/'``
```
Comment (hagenw) [inline CHANGELOG.rst:51]: Maybe better?

```suggestion
* Changed: path on backend must start with ``'/'``
```

File: CHANGELOG.rst [hunk 6]
```diff
@@ -7,6 +7,58 @@ The format is based on `Keep a Changelog`_,
...
+* Changed: version must be non-empty and may not contain invalid characters
+* Changed: option to install only specific backends
+  and their dependencies
+* Removed:
+  ``audbackend.Backend.glob()``,
+  ``audbackend.Backend.path()``
+* Removed: support for ``Python 3.7``
+* Removed: dependency to ``audfactory``
```
Comment (hagenw) [inline CHANGELOG.rst:59]: I'm not sure if `to` is also ok, so I would change to

```suggestion
* Removed: dependency on ``audfactory``
```

**Review by hagenw — COMMENTED** [review]
Great, the changelog reflects quite well, that we spent some effort ;)

File: CHANGELOG.rst (hunk 4 above)
Comment (frankenjoe) [reply CHANGELOG.rst]: yep, fixed

File: CHANGELOG.rst (hunk 3 above)
Comment (frankenjoe) [reply CHANGELOG.rst:41]: fixed

**Pull request merged**